app/classifier/index.cjsx

```diff
@@ -250,17 +250,11 @@ Classifier = React.createClass
     </div>
 
   renderExpertOptions: ->
-    if @props.project?
-      getUserRoles = @props.project.get 'project_roles'
+    if @props.project? && @props.user?
+      getUserRoles = @props.project.get('project_roles', user_id: @props.user.id)
```

Comment:
Adding a query param here means the JSON-API client no longer caches the value locally, so it'll re-request every time the component renders (which is very often). We'll need to cache this value in a property of the component, and clear it when the project or user changes.